Add a v3 version of the document

jimsch opened the pull request:
Add a version of the document that uses the v3 vocabulary and thus can have the svg files included rather than referenced.

jimsch committed:
Add a v3 version of the document

Add a version of the document that uses the v3 vocabulary and thus can have the svg files included rather than referenced.

reschke reviewed (commented):
Some of the xi:includes lack the ".xml" extension. If xml2rfc resolves them anyway, this should be reported as bug.

jimsch committed:
Add extension to all of the hrefs

xi:include should have the full file name and not allow for the extension to be implied.

jimsch commented:
The missing ".xml" extensions have been added in.  xml2rfc does resolve them, this would be because it uses the same name resolution code that has always been used for dealing with entities and <?rfc include?>.  It has been normal for people to omit the extension in these cases.  Go ahead and report if you want, but I don't have any issues with the current behavior myself since I don't use your tool chain

reschke commented:
Understood. It's a problem because the behavior is specified by the XInclude spec - we can't just resolve URIs "correctly" when the server returns 404. This isn't XInclude.

Opened https://trac.tools.ietf.org/tools/xml2rfc/trac/ticket/381
